code/modules/clothing/spacesuits/rig/modules/combat.dm

```diff
@@ -173,7 +173,9 @@
 	if(gun)
 		gun = new gun(src)
 
-/obj/item/rig_module/mounted/engage(atom/target)
+/obj/item/rig_module/mounted/engage(atom/target, skip)
+	if(skip)
```

Comment:
this skip var seems like a dirty hack, whats it all about?

Reply:
Instead of calling proc that is used only for stasis every time someone shoots taser or any other mounted gun we will call it only for stasis